cli/README.md

```diff
@@ -0,0 +1,52 @@
+# lumina-cli
+
+Command line interface for running [Lumina](../README.md) node for the Celestia network either locally or in a browser.
+
+When built with default features, lumina-cli compiles only natively running code. If you want to serve lumina-wasm-node and run it in browser, use `embedded-lumina` feature flag.
+As a shorthand, `lumina` executable can be renamed to `lumina-node`, which will cause it to act as local node only, same as if invoked with `lumina node`.
+
+
+## Building and running
+
+Install common dependencies
+
+```bash
+# install dependencies
+sudo apt-get install -y build-essential curl git protobuf-compiler
+
+# install rust
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+
+# open a new terminal or run
+source "$HOME/.cargo/env"
+
+# clone the repository
+git clone https://github.com/eigerco/lumina
+cd lumina
+```
+
+### Running the node natively
+
+```bash
+# install lumina
+cargo install --path cli
```

Comment:
This is what a user will see on crates.io/crates/lumina-cli, I think we should keep there only the instructions for a installation from `crates.io` directly